alert/topics.go

```diff
@@ -185,24 +191,24 @@ func PatternMatch(pattern, id string) bool {
 }
 
 type Topic struct {
-	id string
-
-	mu sync.RWMutex
-
-	events map[string]*EventState
-	sorted []*EventState
+	id           string
+	mu           sync.RWMutex
+	bufferLength int
+	events       map[string]*EventState
+	sorted       []*EventState
 
 	collected *expvar.Int
 	statsKey  string
 
 	handlers []*bufHandler
 }
 
-func newTopic(id string) *Topic {
+func newTopic(id string, bufferLength int) *Topic {
```

Comment:
Sorry, this was the one I thought should be a method of the Topics struct, so `bufferLength` is magically available.  My mistake on `NewTopics()` being a method of the thing that it created.

Reply:
That's actually a good idea.